test/unit/test_cli.py

```diff
@@ -0,0 +1,191 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests CLI that saves user account to disk."""
+from typing import List
+import unittest
+from unittest.mock import patch
+from textwrap import dedent
+
+from qiskit_ibm_runtime._cli import SaveAccountCLI, select_from_list
+
+from qiskit_ibm_runtime.accounts.account import IBM_CLOUD_API_URL, IBM_QUANTUM_API_URL
+from ..ibm_test_case import IBMTestCase
+
+
+class MockIO:
+    """Mock `input` and `getpass`"""
+
+    # pylint: disable=missing-function-docstring
```

Comment:
Imo this can be set at the file level since it's a test.

Reply:
I'm gonna leave it as I think docstring reminders are still useful to encourage explanations of the tests. We can always ignore them in the future.